doc/cla/corporate/tecnativa.md

```diff
@@ -17,7 +17,6 @@ Rafael Blasco rafael.blasco@tecnativa.com https://github.com/rafaelbn
 Sergio Teruel sergio.teruel@tecnativa.com https://github.com/sergio-teruel
 Carlos Dauden carlos.dauden@tecnativa.com https://github.com/carlosdauden
 Jairo Llopis jairo.llopis@tecnativa.com https://github.com/yajo
-Vicent Cubells vicent.cubells@tecnativa.com https://github.com/cubells
-Luis Montalba luis.montalba@tecnativa.com https://github.com/luismontalba
 David Vidal david.vidal@tecnativa.com https://github.com/chienandalu
-Cristina Martín cristina.martin@tecnativa.com https://github.com/cristinamartinrod
```

Comment:
as Cristina has already made commits, could you keep this line and add a "(up to <date>)" next to it instead?

Reply:
Done